project_from_quotation/models/sale_order.py

```diff
@@ -0,0 +1,188 @@
+# Copyright (C) 2024 Cetmix OÜ
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+from odoo import _, api, exceptions, models
+
+
+class SaleOrder(models.Model):
+    _inherit = "sale.order"
+
+    def _compute_project_ids(self):
+        """
+        Override related projects computation to
+        include projects containing tasks related to
+        order lines of this quotation
+        """
+        super()._compute_project_ids()
+        for order in self:
+            quotation_projects = (
+                self.env["project.task"]
+                .search(
+                    [
+                        ("quotation_line_id", "in", order.order_line.ids),
+                    ]
+                )
+                .mapped("project_id")
+            )
+            order.project_ids |= quotation_projects
+            order.project_count = len(order.project_ids)
```

Comment:
@ivs-cetmix I made comptutation logic of count field as it was, because last solution didn't take into account that some records in `quotation_projects` may be already present in `order.project_ids`. Same for tasks count computation